hera_cal/lstbin.py

```diff
@@ -512,37 +510,34 @@ def lst_bin_arg_parser():
     return a
 
 
-def config_lst_bin_files(data_files, dlst=None, atol=1e-10, lst_start=None, lst_stop=None, fixed_lst_start=False, verbose=True,
-                         ntimes_per_file=60):
+def config_lst_bin_files(data_files, dlst=None, atol=1e-10, lst_start=None, verbose=True, ntimes_per_file=60):
     """
     Configure data for LST binning.
 
-    Make an lst grid, starting LST and output files given
+    Make a 24 hour lst grid, starting LST and output files given
     input data files and LSTbin params.
 
     Parameters
     ----------
-    data_files : type=list of lists: nested set of lists, with each nested list containing
-                 paths to data files from a particular night. These files should be sorted
-                 by ascending Julian Date. Frequency axis of each file must be identical.
+    data_files : type=list of lists: nested set of lists, with each nested list containing paths to
+                 data files from a particular night. Frequency axis of each file must be identical.
     dlst : type=float, LST bin width. If None, will get this from the first file in data_files.
+    atol : type=float, absolute tolerance for LST bin float comparison
     lst_start : type=float, starting LST for binner as it sweeps from lst_start to lst_start + 2pi.
-        Default is first LST of first file.
-    lst_stop : type=float, stopping LST for binner as it sweeps from lst_start to lst_start + 2pi.
-        Default is lst_start + 2pi.
-    fixed_lst_start : type=bool, if True, LST grid starts at lst_start, regardless of LST of first data
-        record. Otherwise, LST grid starts at LST of first data record.
+        Default is first LST of the first file of the first night.
     ntimes_per_file : type=int, number of LST bins in a single output file
 
     Returns
     -------
-    lst_grid : float ndarray holding LST bin centers
+    lst_grid : float ndarray holding LST bin centers. Spans 2 pi radians.
```

Comment:
does it make sense to only return lst_grid indices that are populated by the file_lsts? In other words, we should drop file_lsts that are empty, as well as lst_grid indices that are empty. that makes it closer to the behavior we had before, but I think also accomplishes what you want here, which is a simplification of the creation of the grid.

Reply:
Ok, I've cut the lst_grid to only include lsts that are in the file_lsts